test/integration/fdp-class.spec.ts

```diff
@@ -243,6 +243,25 @@ describe('Fair Data Protocol class', () => {
       expect(directoryInfo.name).toEqual(directoryName)
       expect(directoryInfo1.name).toEqual(directoryName1)
     })
+
+    it('should delete a directory', async () => {
+      const fdp = createFdp()
+      const user = generateUser(fdp)
+      const pod = generateRandomHexString()
+      const directoryName = generateRandomHexString()
+      const directoryFull = '/' + directoryName
+      await topUpAddress(fdp)
+
+      await fdp.account.register(user.username, user.password)
```

Comment:
why is it necessary to register the user?

Reply:
User registration in each individual test is required for independent storage testing

Reply:
but a simple wallet generation wouldn't be enough?

Reply:
Created an issue for that https://github.com/fairDataSociety/fdp-storage/issues/59